### PR TITLE
Added Elgato Stream Deck Neo

### PR DESCRIPTION
**Pull request by mettbroetchen43**
Added Elgato Stream Deck Neo to hwdb

### CODE REVIEW

**Review by GeorgesStavracas — APPROVED** [review]
LGTM. The device model is correct.

This was tested with https://gitlab.gnome.org/World/boatswain/-/merge_requests/37

**Review by GeorgesStavracas — CHANGES_REQUESTED** [review]
Hm actually I just noticed something: please add this to `hwdb.d/70-av-production.hwdb` instead. There is a list of other Stream Deck devices there.

**Review by GeorgesStavracas — APPROVED** [review]
The code looks correct now, please squash the second commit with the first one and it'll be perfect to land!

File: hwdb.d/usb.ids [hunk 1]
```diff
@@ -15920,6 +15920,7 @@
 	0063  Stream Deck Mini
 	006c  Stream Deck XL
 	006d  Stream Deck original V2
+	009a  Stream Deck Neo
```
Comment (yuwata) [inline hwdb.d/usb.ids]: Please drop the change. This file is automatically picked from upstream, so the change will be dropped.

**Review by yuwata — CHANGES_REQUESTED** [review]

File: hwdb.d/usb.ids (hunk 1 above)
Comment (mettbroetchen43) [reply hwdb.d/usb.ids]: Done.

**Review by yuwata — APPROVED** [review]
LGTM.

**Pull request merged**